code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -708,7 +714,8 @@
 		/obj/item/borg/sight/meson,
 		/obj/item/storage/bag/ore/cyborg,
 		/obj/item/pickaxe/drill/cyborg,
-		/obj/item/kinetic_crusher,
+		/obj/item/shovel,
+		/obj/item/kinetic_crusher, //bubber-edit
```

Comment:
Comment on wrong line + modularity

Reply:
The comment is on the line of the bubber edit. The one above is TG code.